changelogs/unreleased/windows_gemfile.md

```diff
@@ -0,0 +1,67 @@
+## Native Gemfile.lock support for Windows Ruby users with Bundler 2.2+
+
+Ruby applications that use Windows and Bundler 2.2+ will no longer have their `Gemfile.lock` deleted and re-generated on deployment. Instead, they will rely on Bundler 2.2+'s support for multiple platforms to correctly resolve dependencies. We recommend any Windows Ruby user to upgrade their Bundler version in the `Gemfile.lock` to Bundler 2.2+ to take advantage of this behavior:
+
+> note
+> The leading `>` indicates a command prompt and should not be copied.
+
+```
+> gem install bundler
+> bundle update --bundler
+> git add Gemfile.lock
+> git commit -m "Upgrade bundler"
+```
+
+This change is reflected in the [Deploying a Ruby Project Generated on Windows](https://devcenter.heroku.com/articles/bundler-windows-gemfile) Devcenter article. More information on the history of the behavior can be found [on this GitHub issue](https://github.com/heroku/heroku-buildpack-ruby/issues/1157).
+
+<!--
+
+https://devcenter.heroku.com/admin/articles/315/edit
+
+# Using Bundler
+
+To use, install bundler run:
+
+> note
+> Commands are prefixed with `>` to indicate they should be run in a command prompt, do not copy the `>` character.
```

Comment:
How is this md rendered? rn this looks like:
> note 
> Commands are prefixed... 

I think we can add a line in between here so it looks like

> note

> Commands are prefixed...

or using a colon
> Note: Commands are prefixed with...
```suggestion
> note

> Commands are prefixed with `>` to indicate they should be run in a command prompt, do not copy the `>` character.
```

Reply:
> How is this md rendered? rn the github md render this looks like:

I'll try to find a stylesheet link, but the DevCenter supports some custom markdown blockquote identifiers. When it sees those, instead of producing a blockquote it styles them according to the first line. For example, in this article https://devcenter.heroku.com/articles/getting-started-with-rails6, the markdown for that callout is: 

```
>note
>Using dynos and databases to complete this tutorial counts towards your usage. We recommend using our [low-cost plans](https://blog.heroku.com/new-low-cost-plans) to complete this tutorial. Eligible students can apply for platform credits through our new [Heroku for GitHub Students program](https://blog.heroku.com/github-student-developer-program).
```